README.md

```diff
@@ -47,6 +47,27 @@ Then run the client, specifying the directory in which your S3 bucket should be
 
 The client will run in the background by default, and the `~/mnt` directory now gives access to the objects in your S3 bucket.
 
+### Mountpoint for Amazon S3 using Docker
+
+First, build the docker image using the below command.
+
+    cd mountpoint-s3/docker
```

Comment:
It's `mountpoint-s3/docker` here but the actual Dockerfile is in `docker`. So, either we move that file to the right path or change the command here.

Reply:
We have offline conversation that we're assuming user are running the command from outside project root directory. It's the same assumption on [how to compile section](https://github.com/awslabs/mountpoint-s3/blob/2f07b5ec976aad219e2d1445909eca3cf0b4976a/README.md?plain=1#L36-L37), so I think this is fine. This is a bit confusing that we also use the same name for sub project.

Reply:
But feel free to suggest the change if you still think we should update it,

Reply:
Thank you Monthonk; I am aligned. Keeping the current version as is.